Automattic/wp-calypso
# Remove Unused Illustrations

#### Changes proposed in this Pull Request

* Removes unused illustrations as part of the attempt to migrate images to Webpack. As advised @bluefuton, this is in a slightly bigger PR compared to doing each one individually!

This will only be for the ones in `static/images/illustrations/`

#### Testing instructions

In all cases, verify that the illustrations are actually unused. I've effectively just searched the repo for them, and looked at them to see if any of them ring a bell, although some may be duplicates.

- advertisingRemoved.svg: https://github.com/Automattic/wp-calypso/search?q=advertisingRemoved.svg&unscoped_q=advertisingRemoved.svg
- anti-spam.svg: https://github.com/Automattic/wp-calypso/search?q=%22spam.svg%22&unscoped_q=%22spam.svg%22
- branding.svg: https://github.com/Automattic/wp-calypso/search?q=%22branding.svg%22&unscoped_q=%22branding.svg%22
- business-address.svg: https://github.com/Automattic/wp-calypso/search?q=business-address&unscoped_q=business-address
- checkEmailsDesktop.svg: https://github.com/Automattic/wp-calypso/search?q=checkEmailsDesktop&unscoped_q=checkEmailsDesktop
- checkEmailsPhone.svg: https://github.com/Automattic/wp-calypso/search?q=checkEmailsPhone&unscoped_q=checkEmailsPhone
- contact-us.svg: https://github.com/Automattic/wp-calypso/search?q=contact-us.svg&unscoped_q=contact-us.svg
- email.svg: https://github.com/Automattic/wp-calypso/search?q=%22email.svg%22&unscoped_q=%22email.svg%22
- expert.svg: https://github.com/Automattic/wp-calypso/search?p=2&q=expert.svg&unscoped_q=expert.svg
- gauge.svg: https://github.com/Automattic/wp-calypso/search?q=%22gauge.svg%22&unscoped_q=%22gauge.svg%22
- google-adwords-stats.svg: https://github.com/Automattic/wp-calypso/search?q=google-adwords-stats.svg&unscoped_q=google-adwords-stats.svg
- homepage-news.svg: https://github.com/Automattic/wp-calypso/search?q=homepage-news&unscoped_q=homepage-news
- homepage-static.svg: https://github.com/Automattic/wp-calypso/search?q=homepage-static&unscoped_q=homepage-static
- illustration-write.svg: https://github.com/Automattic/wp-calypso/search?q=illustration-write&unscoped_q=illustration-write
- jetpack-business-address.svg: https://github.com/Automattic/wp-calypso/search?q=jetpack-business-address&unscoped_q=jetpack-business-address
- jetpack-header.svg: https://github.com/Automattic/wp-calypso/search?q=%22jetpack-header.svg%22&unscoped_q=%22jetpack-header.svg%22
- jetpack-security.svg: https://github.com/Automattic/wp-calypso/search?q=jetpack-security.svg&unscoped_q=jetpack-security.svg
- jetpack-spam.svg: https://github.com/Automattic/wp-calypso/search?q=jetpack-spam&unscoped_q=jetpack-spam
- jetpack-start.svg: https://github.com/Automattic/wp-calypso/search?q=jetpack-start&unscoped_q=jetpack-start
- jetpack-stats.svg: https://github.com/Automattic/wp-calypso/search?q=jetpack-stats&unscoped_q=jetpack-stats
- migrating-host-assisted.svg: https://github.com/Automattic/wp-calypso/search?q=migrating-host-assisted.svg&unscoped_q=migrating-host-assisted.svg
- plugins-big.svg: https://github.com/Automattic/wp-calypso/search?q=plugins-big.svg&unscoped_q=plugins-big.svg
- plugins-small.svg: https://github.com/Automattic/wp-calypso/search?q=plugins-small.svg&unscoped_q=plugins-small.svg
- site-title.svg: https://github.com/Automattic/wp-calypso/search?q=%22site-title.svg%22&unscoped_q=%22site-title.svg%22
- speed.svg: https://github.com/Automattic/wp-calypso/search?q=%22speed.svg%22&unscoped_q=%22speed.svg%22
- upsell-enable.svg: https://github.com/Automattic/wp-calypso/search?q=upsell-enable.svg&unscoped_q=upsell-enable.svg
- upsell-payments.svg: https://github.com/Automattic/wp-calypso/search?q=upsell-payments&unscoped_q=upsell-payments
- video-hosting-13GB.svg: https://github.com/Automattic/wp-calypso/search?q=video-hosting-13G.svg&unscoped_q=video-hosting-13G.svg
- video-hosting-unlimited.svg: https://github.com/Automattic/wp-calypso/search?q=video-hosting-unlimited.svg&unscoped_q=video-hosting-unlimited.svg
- videoAudioPosts.svg: https://github.com/Automattic/wp-calypso/search?q=%22videoAudioPosts.svg%22&unscoped_q=%22videoAudioPosts.svg%22
- waitTime.svg: https://github.com/Automattic/wp-calypso/search?q=waitTime&unscoped_q=waitTime

Part of #36850
